apps/src/templates/progress/progressHelpers.js

```diff
@@ -194,3 +197,87 @@ export const processedLevel = level => {
       level.sublevels && level.sublevels.map(level => processedLevel(level))
   };
 };
+
+const getLevelResult = serverProgress => {
```

Comment:
It looks like `serverProgress` is the same object as `serverObject` on line 222, right?  If so, could we use the same name in the two functions?

Reply:
good idea!